scripts/synapse_port_db

```diff
@@ -629,7 +629,13 @@ class Porter(object):
             await self._setup_state_group_id_seq()
             await self._setup_user_id_seq()
             await self._setup_events_stream_seqs()
-            await self._setup_device_inbox_seq()
+            await self._setup_sequence(
+                "device_inbox_sequence", ("device_inbox", "device_federation_outbox")
+            )
```

Comment:
It seems the other sequences don't *quite* follow the same pattern so have custom functions. It could probably be abstract to handle them, but...not sure it is worth it.